cmd/sytest-coverage/main.go

```diff
@@ -100,6 +87,60 @@ func main() {
 	fmt.Printf("\nTOTAL: %d/%d tests converted\n", numComplementTests, total)
 }
 
+// filenameToTestName and testNameToFilename
+// will filter ignored tests
+func getList() (map[string][]string, map[string]string) {
+	var ignoredTests = make(map[string]bool)
+	var ignoredPaths []string
+	ignoredBody, err := ioutil.ReadFile("./sytest.ignored.list")
+	if err != nil {
+		// ignore error, set body to nothing
+		ignoredBody = []byte{}
+	}
+	ignoredLines := strings.Split(string(ignoredBody), "\n")
+	for _, ignoredLine := range ignoredLines {
+		ignoredLine = strings.TrimSpace(ignoredLine)
+
+		if len(ignoredLine) == 0 || ignoredLine[0] == '#' {
+			continue
+		}
+
+		if ignoredLine[0] == '!' {
+			ignoredPaths = append(ignoredPaths, ignoredLine[1:])
+		}
+
+		ignoredTests[ignoredLine] = true
+	}
+
+	body, err := ioutil.ReadFile("./sytest.list")
+	if err != nil {
+		panic(err)
+	}
+	testLines := strings.Split(string(body), "\n")
+	filenameToTestName := make(map[string][]string)
+	testNameToFilename := make(map[string]string)
+lines:
+	for _, line := range testLines {
+		name, filename := extract(line)
+		if name == "" || filename == "" {
+			continue
+		}
+		if _, ok := ignoredTests[name]; ok {
+			continue
+		}
+		for _, path := range ignoredPaths {
+			if strings.Contains(filename, path) {
```

Comment:
`Contains` probably isn't good enough in the general case as an ignored path of "foo" matches "food" which isn't what we want. Probably fine for now though.